.github/workflows/run_pytest.yml

```diff
@@ -21,6 +21,6 @@ jobs:
           python -m pip install --upgrade pip  
           if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
       - name: Test with pytest
-        run: coverage run -m pytest test/ -v -s
+        run: coverage run --omit='*/dist-packages/*' -m pytest test/ -v -s
```

Comment:
What is the reason for this change?

Reply:
To prevent pandas and matplotlib dependencies from being included in coverage report, which drops coverage percentage way below minimum required.